Tests/XCUITests/ExperimentIntegrationTests.swift

```diff
@@ -0,0 +1,67 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+
+final class ExperimentIntegrationTests: BaseTestCase {
+
```

Comment:
Removing empty line due to swiftlint `Don't include vertical whitespace (empty line) after opening braces`
```suggestion
```

Reply:
Do you have custom settings for swiftlint? Mine didn't say any of these

Reply:
I didn't pull the PR, those are from Bitrise. We don't run with anything special, just need to make sure to be at the root of the project + have latest version of Swiftlint.